src/System.Text.Json/src/System/Text/Json/JsonConstants.cs

```diff
@@ -22,6 +22,13 @@ internal static class JsonConstants
         public const byte BackSpace = (byte)'\b';
         public const byte FormFeed = (byte)'\f';
         public const byte Asterisk = (byte)'*';
+        public const byte Colon = (byte)':';
+        public const byte Minus = (byte)'-';
+        public const byte Period = (byte)'.';
+        public const byte Plus = (byte)'+';
+        public const byte Hyphen = (byte)'-';
```

Comment:
nit: We have both `Hyphen` and `Minus` and both are identical. Have only one?
Apparently, its called Hyphen-Minus (https://www.fileformat.info/info/unicode/char/2d/index.htm), but if we are keeping just one, I would call it `Hyphen`.

Reply:
We'll go with `Hyphen`.